assignments/array-methods.js

```diff
@@ -77,7 +90,20 @@ console.log(ticketPriceTotal);
 // Now that you have used .forEach(), .map(), .filter(), and .reduce().  I want you to think of potential problems you could solve given the data set and the 5k fun run theme.  Try to create and then solve 3 unique problems using one or many of the array methods listed above.
 
 // Problem 1
-
+const largeDonations = runners.filter(word => word.shirt_donations > 90);
```

Comment:
This doesn't work, but I see where you are going with it.

Check out my suggestion:
```suggestion
const largeDonations = runners.filter(word => word.donation > 90);
```
What you are saying with this filter is "go through every object in the array, find all of the objects with a key called shirt_donations and a value greater than 90".

The problem is there are no keys called shirt_donations. There are, however, keys called donations.